content/docs/next-release/deployment.md

```diff
@@ -288,6 +288,64 @@ Shards and replicas are some configuration values to take special attention to,
 index creation. [This article](https://qbox.io/blog/optimizing-elasticsearch-how-many-shards-per-index) goes into
 more information about choosing how many shards should be chosen for optimization.
 
+#### Elasticsearch Rollover
+
+[Elasticsearch rollover](https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-rollover-index.html) is index management strategy that optimizes use of resources allocated to indices.
+For example indices which do not contain any data still allocate shards or in contrary a single index might contain significantly more data than the others.
+Jaeger by default stores data in daily indices which might not optimally utilize resources. Rollover feature can be enabled by `--es.use-aliases=true`.
+
+Rollover allows rolling over to a new index based on different criteria:
+
+* `max_age` - the maximum age of the index
+* `max_docs` - the maximum documents in the index
+* `max_size` - the maximum estimated size of primary shards (since Elasticsearch 6.x)
+
+Rollover index management strategy is more complex than using the default daily indices and it requires an initialisation job to prepare the storage and two cron jobs to manage indices.
+
+To learn more about rollover index management in Jaeger refer to this
+[article](https://medium.com/jaegertracing/using-elasticsearch-rollover-to-manage-indices-8b3d0c77915d).
+
+##### Initialize
+
+The following command prepares Elasticsearch for rollover deployment by creating index aliases, indices and installing Jaeger index templates:
+
+```sh
+docker run -it --rm --net=host jaegertracing/jaeger-es-rollover:latest init http://localhost:9200 # <1>
+```
+
+<1> Add `-e ARCHIVE=true` to initialize archive storage
+
+After the initialization Jaeger can be deployed with `--es.use-aliases=true`.
+
+##### Rolling over to a new index
+
+The next step is to periodically execute rollover API which rolls the write alias to a new index based on supplied conditions. The command also adds a new index to read alias to make new data available for search.
```

Comment:
periodically execute the rollover API...
The command also adds a new index to the read alias....